src/dimension.h

```diff
@@ -64,15 +64,17 @@ typedef struct Point
 #define POINT_SIZE(cardinality)							\
 	(sizeof(Point) + (sizeof(int64) * (cardinality)))
 
+#define DEFAULT_CHUNK_TIME_INTERVAL (USECS_PER_DAY * 30)	/* 1 month */
```

Comment:
I wonder if now is a good time to switch from 30 days to 1 day or something smaller?

Reply:
Maybe, but perhaps we should do this in a separate PR to make the switch more explicit?

Reply:
Def a separate PR would be preferred IMHO